lms/djangoapps/discussion_api/api.py

```diff
@@ -697,6 +696,30 @@ def update_comment(request, comment_id, update_data):
     return api_comment
 
 
+def get_thread(request, thread_id):
+    """
+    Retrieve a thread.
+
+    Arguments:
+
+        request: The django request object used for build_absolute_uri and
+          determining the requesting user.
+
+        thread_id: The id for the thread to retrieve
+
+    Raises:
+
+        PermissionDenied: if user does not have permission to view thread
+
+    """
+    cc_thread, context = _get_thread_and_context(request, thread_id)
+    if has_permission(cc_thread, context):
```

Comment:
One thing I'd like to confirm. We would like anyone who can view this thread to be able to call `get_thread`, but it looks like `has_permission` will only be `True` if the user is a "privileged user" or author, which does not include most users, who are only able to view the thread but not modify it. Right?